test/NuGet.Core.FuncTests/Dotnet.Integration.Test/ProjectPropertyTests.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using FluentAssertions;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace Dotnet.Integration.Test
+{
+    [Collection(DotnetIntegrationCollection.Name)]
+    public class ProjectPropertyTests
```

Comment:
Maybe just use this fixture since I don't think you need what's in this Collection?

```suggestion
    public class ProjectPropertyTests : ICollectionFixture<MsbuildIntegrationTestFixture>,
```

Reply:
Is it possible to use `ICollectionFixture` without `CollectionAttribute`?  Or did you typo, and mean `IClassFixture`?

But yes, `MsbuildIntegrationTestFixture` is very expensive to setup and cleanup (mostly setup), which is why all tests in the project use the collection, so it only has to be done once. Unfortunately, this also means that tests don't run in parallel, so maybe we should investigate alternatives (a static class), but for now this is how we copy the .NET SDK and patch it with the current build of NuGet.